### ISSUE TITLE
Setup synology

### ISSUE THREAD

**Issue by technicalpickles** [technicalpickles/picklehome#75]
The drives are connected, but need to be allocated. There's 6 4TB drives. I'm thinking to use SHR-2, which leaves 16TB. I think it makes sense to start with some reasonable amount for each purpose, like 1TB, and expand them as needed, rather than allocating to one volume.

It'd be good to get it going over SSL too (https://stefandingemanse.nl/how-to-use-lets-encrypt-ssl-certificate-on-synology-dsm/)

cc https://github.com/technicalpickles/picklehome/issues/74

**Comment by technicalpickles** [comment]
I made one storage pool, and a few volumes on it:

- channels dvr (2T)
- media (2T)
- backups (2T)

I created a shared folder on the media one for https://github.com/technicalpickles/picklehome/issues/74

I looked a little more into SSL and it seems like it will be annoying to use letsencrypt for, since it'd need to be internet facing. I could do a self-signed one, but I got a little confused when it was trying to make two that couldn't be the same (turned out one was root and other was server).

**Comment by technicalpickles** [comment]
Got some tips on setting up LetEncrypt:

- use DNS based registration
- run it centralized somewhere (ie on the nuck)
- have post hook scripts upload to synology or wherever

I'm told the you need to manually upload from to the synology interface the first time, but after that they will be in `/usr/syno/etc/certificate/_archive/OpytNe` (where `OpytNe` is a random string it generates for you).

**Comment by technicalpickles** [comment]
Some more resources:

- https://synoguide.com/2016/04/14/secure-your-synology-with-https-ssl-certificate-from-lets-encrypt/
- https://synoguide.com/2016/04/14/secure-your-synology-with-https-ssl-certificate-from-lets-encrypt/
- https://community.ubnt.com/t5/UniFi-Routing-Switching/USG-DNS-server/td-p/2182469

I ended up creating registering a new domain for internal use, and making a wildcard for that. I used Unifi SDN to set that domain as the default lookup, and made sure a client name was set for the Synology, so I could access it as `picklestation.whatever`. Then I uploaded the certs to the synology as described in the above links.

Found this README in the letsencrypt folder useful:

```

`[cert name]/privkey.pem`  : the private key for your certificate.
`[cert name]/fullchain.pem`: the certificate file used in most server software.
`[cert name]/chain.pem`    : used for OCSP stapling in Nginx >=1.3.7.
`[cert name]/cert.pem`     : will break many server configurations, and should not be used
                 without reading further documentation (see link below).
```
